### PR TITLE
add fix to typos and grammar

### LINKED ISSUE #185: Various typos and grammar
Below is a list of some typos myself and @HedgehogHolly have found 

Holly: I used the Grammarly website to pick up a lot of these

**Home**
- [x] practise -> practice
- [x] sous chef -> sous-chef

**What we're doing**
- [x] their situations and stories -> their situations and stories.
- [x] confusing,overwhelming -> confusing, overwhelming

**Harry's Bio**
- [x] the latter, and -> the latter and

**Charlie's Bio** 
- [x] ,. -> .
- [x] directing a SME focused IT services company -> directing an SME focused IT services company

**Matt's Bio**
- [x] by the time you read this I probably -> by the time you read this I will probably

**Toby's Bio**
- [x] I'm hold an -> I hold an

**Neil**
- [x] Cyber security -> Cybersecurity
- [x] Fortunately times have moved on ->  Fortunately, times have moved on 
- [x] As some light relief I take scouts -> As some light relief, I take scouts
- [x] my hobby of live action -> my hobby of live-action 
- [x] role play runs smoothly -> role-play runs smoothly

**Katja**
- [x] vim and linux -> Vim and Linux
- [x] community, open source, transparency -> community, open-source, transparency
- [x]  coding as a kid, and ->  coding as a kid and
- [x] at the BBC, and -> at the BBC and

**Karl**
- [x]  my work life balance  ->  my work-life balance 
- [x] child care/school runs etc.-> childcare/school runs etc.
- [x] Outside of the office -> Outside of the office,

**Andy**
- [x] Neontribe also sponsor me studying a degree -> Neontribe also sponsors my degree in Computer Science 

**Kara**
- [x] so worked in -> so I worked in
- [x] self taught SQL and ended -> self-taught SQL and ended 
- [x]  process related; as ->  process-related; as

**Rose**
- [x] During my time in the sector I was -> During my time in the sector, I was

**Kat**
- [x] 15 year career -> 15-year career
- [x] Can frequently be found doing game design -> I can frequently be found doing game design

**Nick**
- [x]  game development company, ->  game development company

**Rob**
- [x] College cancelled the -> The college cancelled the

### PR DESCRIPTION
**Pull request by notpluto**
Fixes #185 

## Description

- made suggested changes

### CODE REVIEW

File: src/data/tribers/andy.md [hunk 1]
```diff
@@ -7,4 +7,4 @@ words:
...
 ---
 
-I was adopted by Neontribe in 2012, rescued from redundancy. I have been working in the industry for two and a half decades. Neontribe also sponsor me studying a degree in Computer Science with the Open University. I love working on tech-for-good projects and building infrastructure from open source components. Working with the other members of the tribe opened my eyes to working in an environment that fosters growth and teamwork. When I'm not working I'm Interested in cooking, gaming and slouching. I am a host for the Lave Radio podcast, and occasionally can be found streaming on Twitch.
+I was adopted by Neontribe in 2012, rescued from redundancy. I have been working in the industry for two and a half decades. Neontribe also sponsors my degree in Computer Science with the Open University. I love working on tech-for-good projects and building infrastructure from open source components. Working with the other members of the tribe opened my eyes to working in an environment that fosters growth and teamwork. When I'm not working I'm Interested in cooking, gaming and slouching. I am a host for the Lave Radio podcast, and occasionally can be found streaming on Twitch.
```
Comment (katjam) [inline src/data/tribers/andy.md]: Should this also be open-source as per `katja.md:11` ?

**Comment by RpprRoger** [comment]
Thanks @notpluto

**Comment by RpprRoger** [comment]
@notpluto How would you feel about having your username listed in a humans.txt? For when we consider #203

**Comment by notpluto** [comment]
Hey @RpprRoger! I would be elated to be considered on the list.

Thank you all for your support. You guys are awesome 😄

**Pull request merged**